src/components/modals/SubmitTranscriptModal.tsx

```diff
@@ -83,7 +83,7 @@ const SubmitTranscriptModal = ({ submitState, onClose }: Props) => {
             {isError && !isLoading && (
               <Box>
                 <Text color="gray.400" fontWeight={500}>
-                  Unsucessful
+                  {err?.response?.data?.message || "Unsuccessful"}
```

Comment:
err is returned as Error or null.

read it as `err?.message` instead

Reply:
So the issue here is Axios extends the  in built error object when returning  the error .when catching the error
so hence any where there is  err?.message on the client side it returns undefined